corehq/messaging/scheduling/scheduling_partitioned/tests/test_dbaccessors_partitioned.py

```diff
@@ -24,11 +24,16 @@
 from corehq.sql_db.tests.utils import DefaultShardingTestConfigMixIn
 from datetime import datetime, date
 from django.test import TestCase
+from django.utils.functional import classproperty
 
 
 @only_run_with_partitioned_database
 class BaseSchedulingPartitionedDBAccessorsTest(DefaultShardingTestConfigMixIn, TestCase):
 
+    @classproperty
```

Comment:
Why does this one need to be decorated? Based on the rest of this PR, I'm assuming that it's "Test" as a prefix on a class name that makes pytest think it's a test, but `BaseSchedulingPartitionedDBAccessorsTest` doesn't start with "Test."

Reply:
Any subclass of `TestCase` is considered to be a test. This one is problematic because it's an abstract base class that doesn't define any tests, and pytest gets cranky about test classes that don't have any tests.